apps/okd4-this-or-that-apps.yaml

```diff
@@ -0,0 +1,24 @@
+repository: https://github.com/baloise-incubator/this-or-that-deployment.git
+#argocdNamespace: "argocd"
+#server: "https://kubernetes.default.svc"
+namespace:
+  annotations: {}
+  labels:
+    client: philipp
+
+enableDefaultNetworkPolicy: false
+enableDefaultLimitRange: false
+allowAllNamespaces: true
+
+roleBindings:
+#  - groupName: "team1"
+#    clusterRoleName: "admin"
+
+clusterResourceWhitelist:
+
+namespaceResourceBlacklist:
+#  - group: 'apps'
+#    kind: 'DaemonSet'
+
+applications:
```

Comment:
`applications` should be managed through the tekton job - not entered manually

Reply:
done